nemo/collections/llm/gpt/model/megatron/hyena/hyena_mixer.py

```diff
@@ -160,7 +162,15 @@ def __init__(
 
         hyena_proj_groups = self.proj_groups if not self.grouped_attention else 1
         grouped_proj_size = self.hidden_size_per_partition // hyena_proj_groups
-        self.hyena_proj_conv = ParallelCausalDepthwiseConv1d(
+        hyena_proj_conv_class = ParallelCausalDepthwiseConv1d
+        if with_state_p:
```

Comment:
IMO let's get rid of this since it's hard-coded to True. That was prob an implementation hack for verifying things.